limited/shepherd

```diff
@@ -1,13 +1,10 @@
 **Shepherd** | Solo Power - Flock Team | Limited
 __Basics__
-Each night, the Shepherd may choose one player to herd. When everyone is dead, herded or part of the flock the Shepherd wins.
+Each night, the Shepherd may choose to tag two players, if two tagged players visit each other they are herded. When everyone is dead, herded or part of the flock the Shepherd wins.
 __Details__
-Whenever a player is herded, they will join a secret channel with the other herded players. Herding a player is an immediate effect. The Shepherd may see everything said in this channel, from the safety of their secret channel. Members of the flock can not be herded.
-
+When a tagged player visits another tagged player they become herded but remain tagged, if they then visit or are visited by another tagged player who is not herded then that player is then also herded.
+Whenever a player is herded, they will join a secret channel with the other herded players. Herding a player is an immediate effect. The Shepherd may see everything said in this channel, from the safety of their secret channel. Members of the flock can not be tagged or herded.
```

Comment:
```suggestion
Whenever a player is herded, they will join a secret channel with the other herded players. Herding a player is an immediate effect. The Shepherd may see everything said in this channel, from the safety of their secret channel. 
Members of the flock can not be tagged or herded.
```
New line since it's a new topic.
Is it necessary to say it's an immediate effect? It seems more like a consequence of the visiting action, so follow whatever timing that action has.

Reply:
I was supposed to change "hearding is immediate" to "tagging is immediate", my bad 😖